discord/commands.py

```diff
@@ -99,6 +99,7 @@ async def __call__(self, data, channel: Optional[Messageable] = None) -> Interac
             'channel_id': str(channel.id),
             'data': data,
             'nonce': str(time_snowflake(datetime.utcnow())),
+            'session_id': state.session_id or _generate_session_id(),
```

Comment:
Learnt something new, I didn't know that `or` could be used to call something else if the first argument was None.